src/applications/mhv/secure-messaging/containers/App.jsx

```diff
@@ -35,7 +35,7 @@ const App = () => {
           {/* toggle log in and out state without using va.gov sign in, to show the toggle button, comment out the visibility:hidden style */}
           <button
             style={{
-              visibility: 'hidden',
+              //  visibility: 'hidden',
```

Comment:
I believe we need to keep this commented out for now and prevent it from making to main branch